english/109_jumla.md

```diff
@@ -57,61 +57,49 @@ Also two personal pronouns can be combined with **si** and **no**.
 – I'm not you.
 
 
-### With adjective subject
+### With adjective or verb subject
 
-The same rule applies also for two adjectives.
+The same rule applies also for two adjectives, or a verb and an adjective.
 
 **jovan si sundar.**
 – Young is beautiful.  
 **nove no puran.**
 – New is not old.  
+**chante si bon.**
+– Singing is good.
 
-### With verb subject
 
-A verb can't be subject as such.
-So the verb is first, use **de** to start a comment about the verb.
+## Action Sentences
 
-**cante si bon.**
-– Singing is good. (Singing is good.)
+A predicate can also be a verb. In this case, there may be objects as well as the subject.
+These come after the predicate.
 
+**mi marche.**
+– I walk.  
+**tu yam yo apel.**
+– You eat some apples.  
+**mimen dona ya pesa.**
```

Comment:
This looks like a pivot structure, i.e. **pesa** could be interpreted as a verb, knowing that 'to give' means also 'to allow' in several languages.

So this sentence should be **mimen dona ya yo pesa** or **mimen dona yo pesa do ya**.

Reply:
you're rite; good point.